tests/test_halo_sampler.py

```diff
@@ -34,7 +34,8 @@
 @pytest.mark.parametrize("cond_type", ["cat", "grid"])
 @pytest.mark.parametrize("cond", range(len(options_delta)))
 def test_sampler(name, cond, cond_type, plt):
-    redshift, kwargs = cint.OPTIONS_HMF[name]
+    _, kwargs = cint.OPTIONS_HMF[name]
+    redshift = 7.0  # we need the largest condition mass to not be >7 sigma
```

Comment:
Why does the largest condition mass need to be <7? Because it will trigger the rare halo case?

Reply:
Yeah, if the condition mass trips the rare halo check it's not a very useful test